querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java

```diff
@@ -30,7 +30,7 @@
 import com.querydsl.sql.codegen.support.PrimaryKeyData;
 
 /**
- * ExtendedBeanSerialzier outputs primary key based equals, hashCode and toString implementations
+ * ExtendedBeanSerializer outputs primary key based equals, hashCode and toString implementations
```

Comment:
I think you could start with the current changed references yes.
@timowest what do you think? Should we change the current changed references, or in another PR? Or do you think it is not necessary?